packages/typegpu/tests/accessor.test.ts

```diff
@@ -170,11 +170,26 @@ describe('tgpu.accessor', () => {
           var color2 = redUniform;
           var color3 = getColor();
 
-          var colorX = vec3f(1, 0, 0).x;
+          var colorX = 1;
```

Comment:
Accessors populate with a constant value are not known at comp-time, which makes them even more useful 🎉